src/background/services/background.ts

```diff
@@ -31,6 +31,9 @@ export class Background {
   ) {}
 
   async start() {
+    // Reset out_of_funds state, we'll detect latest state as we make a payment.
+    await this.storage.setState({ out_of_funds: false })
+
```

Comment:
This will get called only on background start. Fine if background idles after some time, but not if it stays alive. Need a way to get out of out_of_funds when recurring balance renews!

Reply:
If continuous payments are enabled, we can make a payment and know we have funds now. But not without it.

Reply:
Also, TODO: reset the `out_of_funds` on first payment if it was already not reset (don't want to call it on each payment).